doc/method_documentation.rdoc

```diff
@@ -181,3 +181,26 @@ of the method.
 Methods are documented using RDoc syntax.  See the
 {RDoc Markup Reference}[https://docs.ruby-lang.org/en/master/RDoc/Markup.html#class-RDoc::Markup-label-RDoc+Markup+Reference]
 for more information on formatting with RDoc syntax.
+
+=== Code Blocks
+
+A code block should be preceded and followed by a blank line.
+This is unnecessary for the HTML output, but helps in the ri output.
+
+Consider whether <tt>=> ...</tt> in successive lines should be aligned.
```

Comment:
```suggestion
Consider whether comments (e. g. <tt># => ...</tt>) in successive lines should be aligned.
```

Reply:
Have changed to "Consider whether <tt># => ...</tt> in successive codeblock lines should be aligned."